css/style.css

```diff
@@ -1,5 +1,9 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
 
+* {
+  left: 0;
+}
+
 body {
   width: 375px;
   height: 100%;
```

Comment:
- [ ] One of the core requirements of this project is to create a responsive design. Although you have done a great job achieving that but this exact width of `375px` is making it impossible to achieve the responsive design. As soon as I bring the width less than `768px`, all of the content of your website get shrunk to 375px and aligned to the left side of the webpage. Our design should be responsive up to `375px`, so  kindly consider making your design responsive.
  -  Minimum mobile screen size should be 375px. Errors in screen sizes less than 375px should be ignored. Desktop version should be fully responsive.
your design
![ghj](https://user-images.githubusercontent.com/96683843/213529402-ca0ba675-8d38-46e8-838d-1634c61aec13.jpg)